static/css/restyle/restyle.less

```diff
@@ -2050,3 +2050,51 @@ body.editor-tools {
   }
 
 }
+
+/* Source/Diff viewer */
+body.file-viewer {
+  pre, code, kbd, tt, samp, tt {
+    font-family: "andale mono", monospace !important;
```

Comment:
Yep, the hope is that this should be a short-lived band-aid since the plan is to rework the entire front-end in the not too distant.